annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,6 +1,5 @@
 spring.batch.job.enabled=false
-genomenexus.hgvs=<HOST>/hgvs/
-genomenexus.xrefs=<HOST>/xrefs/
+genomenexus.base=<HOST>
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.enrichment_fields=hotspots,mutation_assessor
+genomenexus.enrichment_fields=annotation_summary
```

Comment:
now we only rely on the annotation_summary (field name is subject to change)

Reply:
Default value makes sense.

Although it is good to have some error handling, I wouldn't worry much about invalid enrichment fields. Genome Nexus just ignores the field if it is not valid. Also, currently we don't really need any other field than `annotation_summary`.

Reply:
👌